spec/defines/nodejs_npm_spec.rb

```diff
@@ -23,348 +21,317 @@
 
     # su user npm install <package>
     context 'with package set to express, user set to foo and target set to /home/npm/packages' do
-      let (:params) {{
-        :package => 'express',
-        :target  => '/home/npm/packages',
-        :user    => 'foo',
-      }}
+      let :params do
+        {
+          :package => 'express',
+          :target  => '/home/npm/packages',
+          :user    => 'foo',
+        }
+      end
 
       it 'the npm install command should run under user foo' do
-        is_expected.to contain_exec('npm_install_express').with({
-          'command' => '/usr/bin/npm install express ',
-          'user'    => 'foo',
-         })
+        is_expected.to contain_exec('npm_install_express').with('command' => '/usr/bin/npm install express ',
+                                                                'user'    => 'foo',)
```

Comment:
I vastly prefer the multiline-version too. Much easier to read and no annoying scroll off.